docs/guides/vault-performance-replication.md

```diff
@@ -0,0 +1,44 @@
+---
+subcategory: ""
+page_title: "Configure Vault performance replication - HCP Provider"
+description: |-
+    Configure performance replication between two Plus tier clusters.
+---
+
+# Configure Vault performance replication
+
+Admins and Contributors can use the provider to create Plus tier clusters with Vault [performance replication](https://learn.hashicorp.com/tutorials/cloud/vault-replication) enabled via the `primary_link` parameter. In addition to both clusters being in the Plus tier, both must be of the same size (S, M, L).
+
+Although the clusters may reside in the same HVN, it is more likely that you will want to station your performance replication secondary in a different region, and therefore HVN, than your primary. When establishing performance replication links between clusters in different HVNs, an HVN peering connection is required. This can be defined explicitly using an [`hcp_hvn_peering_connection`](../resources/hvn_peering_connection.md), or HCP will create the connection automatically (peering connections can be imported after creation using [terraform import](https://www.terraform.io/cli/import)). Note HVN peering [CIDR block requirements](https://cloud.hashicorp.com/docs/hcp/network/routes#cidr-block-requirements).
```

Comment:
Is it worth noting as a callout that Plus tier Vault clusters can't be scaled?

Reply:
Probably. This is only going to be true for a short while though :)

Reply:
Added